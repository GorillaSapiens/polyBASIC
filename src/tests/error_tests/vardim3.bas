a := 2
a(4,5) := 3
