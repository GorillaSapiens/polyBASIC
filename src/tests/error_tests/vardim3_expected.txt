SOURCE 2:8, VARIABLE ❮a❯ DEFINED AS NONARRAY ON LINE 1
